tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1338,7 +1348,36 @@ cons_unroll_count_metadata(int unroll_factor)
   unroll= ll_get_md_node(cpu_llvm_module, LL_PlainMDNode, lvcomp, 2);
   return unroll;
 }
+static LL_MDRef
```

Comment:
i removed that functions and instead of them i decided to do everything in this piece of code
`if(BIH_VECTORIZE_WIDTH_ENABLED(bih)) {
          if (LL_MDREF_IS_NULL(loop_md)){
            loop_md = cons_loop_id_md();
          }
          LL_MDRef vectorlength_enable = cons_vectorize_metadata();
          ll_extend_md_node(cpu_llvm_module, loop_md, vectorlength_enable);
          LL_MDRef vectorlength_scalable  = cons_vectorlength_scalable_metadata(BIH_VECTORIZE_WIDTH_SCALABLE( bih )>0);
          ll_extend_md_node(cpu_llvm_module, loop_md, vectorlength_scalable);
          if(BIH_VECTORIZE_WIDTH_FACTOR( bih) && vectorlength_factor > 0) {
            LL_MDRef lvcomp[2];
            LL_MDRef width;
            lvcomp[0] = ll_get_md_string( cpu_llvm_module, "llvm.loop.vectorize.width");
            lvcomp[1] = ll_get_md_i32(cpu_llvm_module, vectorlength_factor);
            width = ll_get_md_node(cpu_llvm_module, LL_PlainMDNode, lvcomp, 2);
            ll_extend_md_node(cpu_llvm_module, loop_md, width);
          }
        }`
-->fixed